utils/patch_helper.py

```diff
@@ -0,0 +1,56 @@
+"""
+Pass in a list of PR:
+`python utils/patch
```

Comment:
I would be useful for this to explain a little more baout what it does i.e. prepares commands for cherry-picking